app/client/src/git/sagas/fetchGitMetadataSaga.ts

```
@@ -24,12 +26,19 @@ export default function* fetchGitMetadataSaga(
         }),
       );
     }
-  } catch (error) {
-    yield put(
-      gitArtifactActions.fetchGitMetadataError({
-        ...basePayload,
-        error: error as string,
-      }),
-    );
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)